create.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body class="app-grid h-500">
+    <header class="p-auto bg-hotpink">CREATE</header>
+    <main class="app-grid__scrollarea bg-blue">
+      <div class="h-100vh">
+        <form>
+          <fieldset>
+            <legend>Add a question</legend>
+            <label for="add-question" aria-label="add-question"></label>
```

Comment:
Check if you need three `fieldset `elements here or if you really need the `fieldset` element in general.
Maybe the `form` element is enough to group the content and you can use the `label` element instead of the `legend` element to title the three `textarea` elements.